CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/aggregators/MaxScoreAggregator.java

```diff
@@ -49,43 +46,16 @@ public ScoreResult aggregate(final List<ScoreResult> results) {
         aggregateResult.setIgnoreNullResults(results.stream()
                 .anyMatch(result -> result.getIgnoreNullResults()));
 
-        final Set<ThreeTuple<GraphElementType, Integer, String>> elements = new HashSet<>();
-        for (final ScoreResult result : results) {
-            for (final ElementScore score : result.get()) {
-                elements.add(ThreeTuple.create(score.getElementType(), score.getElementId(), score.getIdentifier()));
-            }
-        }
-
-        boolean isNull;
-        final List<Float> scores = new ArrayList<>();
-        for (final ThreeTuple<GraphElementType, Integer, String> element : elements) {
-            final GraphElementType type = element.getFirst();
-            final int id = element.getSecond();
-            final String identifier = element.getThird();
-            isNull = false;
-            scores.clear();
-
-            for (final ScoreResult result : results) {
-                result.setIgnoreNullResults(false);
-                for (final ElementScore score : result.get()) {
-                    if (type == score.getElementType() && id == score.getElementId()) {
-                        for (final String scoreName : score.getNamedScores().keySet()) {
-                            isNull |= score.isNull();
-                            scores.add(score.getNamedScore(scoreName));
-                        }
-                    }
-                }
-            }
-
+        results.forEach(scoreResult -> combinedResults.combine(scoreResult));
+        combinedResults.getResult().forEach((IdentificationData key, ElementScore value) -> {
             final Map<String, Float> aggregateScores = new HashMap<>();
-            aggregateScores.put(SCORE_NAME, scores.stream().reduce(Math::max).orElse((float) 0.0));
-
-            aggregateResult.add(new ElementScore(type, id, identifier, isNull, aggregateScores));
-        }
-
+            aggregateScores.put(SCORE_NAME, value.getNamedScores().values().stream().reduce(Math::max).orElse((float) 0.0));
```

Comment:
noting you didn't add this but if you're able to change `(float) 0.0` to `0.0f`, that would be great